coalib/misc/Caching.py

```diff
@@ -1,8 +1,8 @@
 import time
 import os
+import logging
```

Comment:
That makes more sense. We can onboard newcomer as well as it might go out of scope for this PR, and this one is already big enough, I guess.